### PR TITLE
Sandeep

### PR DESCRIPTION
**Pull request by sandeepkrishnagopalappa**
changes

### CODE REVIEW

File: foo/test/__init__.py [hunk 1]
```diff
@@ -1,4 +1,2 @@
 import pytest
 from config import *
```
Comment (rohsmiles) [inline foo/test/__init__.py:2]: This is good, can you try not to import anything here.
Import somewhere else other than test cases.

File: foo/test/test_pom.py [hunk 2]
```diff
@@ -1,28 +1,37 @@
-""" This is the page where all the functions where called from here """
-from foo import *
+""" This is the page where all the functions where called from """
+import pytest
+from config import url,expected_tv_page_title,expected_title,expected_mat_page_title
```
Comment (rohsmiles) [inline foo/test/test_pom.py:3]: This comment is repetitive.
Test case should have only pytest as import statement.

File: foo/test/test_pom.py [hunk 3]
```diff
@@ -1,28 +1,37 @@
...
 
-@pytest.mark.usefixtures('instance')
+@pytest.mark.usefixtures("driver_init","url_init","instance")
```
Comment (rohsmiles) [inline foo/test/test_pom.py:5]: "instance" fixture is enough, please check the fixture workflow.

File: foo/test/test_pom.py [hunk 4]
```diff
@@ -1,28 +1,37 @@
...
-        request.cls.tv.sorting_element_hover()
-        max_price_, element_after ,tv_name_after = request.cls.tv.get_names_price()
+        max_price, element_before ,tv_name_before = self.tv.get_names_price()
+        self.tv.sorting_element_hover()
+        try:
+            self.tv.close_pop_up()
+        except Exception as e:
+            print(e)
+        max_price_, element_after ,tv_name_after = self.tv.get_names_price()
         assert max_price ==  element_after ," The page after sorting is not sorted properly in tv page "
```
Comment (rohsmiles) [inline foo/test/test_pom.py:22]: Till this step it is one test case.
Below this step it should be another test case.

**Review by sandeepkrishnagopalappa — COMMENTED** [review]
committed by solving  the above comments

**Comment by sandeepkrishnagopalappa** [comment]
merged by solving comments

File: foo/test/test_pom.py (hunk 2 above)
Comment (sandeepkrishnagopalappa) [reply foo/test/test_pom.py:3]: In order not to import config module in tests module, i have created a class in config file and 
i have moved all the urls and paths inside the class, Also i have created an instance of that class in conftest file.
With the help of fixtures i have used config file variables in test module .

File: foo/test/test_pom.py (hunk 3 above)
Comment (sandeepkrishnagopalappa) [reply foo/test/test_pom.py:5]: Gone through the fixture work flow and inherited only instance fixture as instance fixture is inheriting all the 
above fixtures.

File: foo/test/test_pom.py (hunk 4 above)
Comment (sandeepkrishnagopalappa) [reply foo/test/test_pom.py:22]: In order to differenciate between the methods of different pages in test file i have made a comment as"Matress Page method".

File: foo/test/__init__.py (hunk 1 above)
Comment (sandeepkrishnagopalappa) [reply foo/test/__init__.py:2]: I have not imported anything in test.py/__init__.py

**Pull request closed without merging**